src/main/java/net/mrscauthd/beyond_earth/events/Gravity.java

```diff
@@ -15,41 +15,45 @@ public static void Gravity(LivingEntity entity, GravityType type, Level world) {
         double venus = 0.04;
         double orbit = 0.02;
 
+        if (Methods.isWorld(world, Methods.overworld)) {
+            return;
+        }
+
         if (Methods.isWorld(world, Methods.moon)) {
             gravityMath(type, entity, moon, -2.5f);
         }
 
-        if (Methods.isWorld(world, Methods.mars)) {
+        else if (Methods.isWorld(world, Methods.mars)) {
             gravityMath(type, entity, mars, -2.0f);
```

Comment:
`world` cannot be more than one type simultaneously.

If we know the world is moon, we don't need to check if that same world is also mars as it is impossible.